packages/tools/client-debug-view/src/components/AudienceView.tsx

```diff
@@ -166,16 +166,34 @@ function HistoryView(props: HistoryViewProps): React.ReactElement {
 		const changeTimeStamp = new Date(changeEntry.timestamp);
 		const wasChangeToday = nowTimeStamp.getDate() === changeTimeStamp.getDate();
 
+        const accordianBackgroundColor: IStackItemStyles = {
+            root: {
+                background: changeEntry.changeKind === "added" ? "lightgreen" : "orange",
+                padding: 3
+            }
+        }
+
 		historyViews.push(
-			<li key={`${changeEntry.clientId}-${changeEntry.changeKind}`}>
-				<b>Client ID: </b>
-				{changeEntry.clientId}
-				<br />
-				<b>Time: </b>{" "}
-				{wasChangeToday ? changeTimeStamp.toTimeString() : changeTimeStamp.toDateString()}
-				<br />
-				<b>Type: </b> {changeEntry.changeKind}
-			</li>,
+            <Stack horizontal={true} styles={accordianBackgroundColor}>
+                <StackItem>
```

Comment:
Nit: If possible, can we center the icon vertically in the space? I'm betting there is a way to do it with styles.

Reply:
<img width="283" alt="image" src="https://user-images.githubusercontent.com/111468570/206372852-27736c8a-78c4-4997-96ab-c9ea783bc6bb.png">

Changes Applied :D